share/commands/remote-run

```diff
@@ -29,10 +29,36 @@
 # @man *-m*: run the agent with multiline output
 # @man +
 # @man *-D*: define a class for this run
+# @man +
+# @man *-a*: run the agent on all known nodes
+# @man +
+# @man *-g*: run the agent on all nodes of the given group
+# @man +
+# @man *-j*: run this number of jobs in parallel
+# @man +
+# @man *-t*: provide an alternate token for group query (default from ~/.rudder)
+# @man +
+# @man *-u*: provide an alternate url for group query (default from ~/.rudder)
+# @man +
+# @man *-C*: provide an alternate config section in ~/.rudder for group query (default to first found)
+
 
 . "${BASEDIR}/../lib/cfengine_parsing.sh"
+. "${BASEDIR}/../lib/api_call.sh"
+
+PORT=5309
+LOOP="xargs -I{}"
 
-while getopts "iIvdqwrRmcb:D:" opt; do
+# Check for js presence
+need_jq() {
+  if ! type jq >/dev/null 2>/dev/null
+  then
+    echo "ERROR: 'jq' is needed to query hosts from server" 1>&2
```

Comment:
Could you make this error message the same as the one below? So : "ERROR: 'jq' must be installed to query hosts from server"

It just makes it a bit clearer :)